### PR TITLE
Update List.js - Removed dead third party hosts

### PR DESCRIPTION
**Pull request by HardenedSteel**
Some third party hosts were not working so I removed.

### CODE REVIEW

File: src/components/Directory/List.js [hunk 1]
```diff
@@ -1435,13 +1400,6 @@ export const list = [
     description:
       "Roverba offers cloud hosting and telecom services to businesses. Address: 7 rue des Maraichers, 69120 Vaulx-en-Velin."
   },
-  {
```
Comment (pavlenex) [inline src/components/Directory/List.js:1438]: This one seems to be still active?

Comment (HardenedSteel) [reply src/components/Directory/List.js:1438]: I guess I thought its offline because the site doesn't support TLS. Also

Comment (pavlenex) [reply src/components/Directory/List.js:1438]: Should we bring them back and remove the rest, wdyt?

Comment (HardenedSteel) [reply src/components/Directory/List.js]: brought back, done

**Pull request merged**